kw/platform/requests_utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import wrapt
+
+
+APP_NAME = os.getenv("APP_NAME")
+PACKAGE_VERSION = os.getenv("PACKAGE_VERSION", "1.0.0")
+ENVIRONMENT = os.getenv("ENVIRONMENT", "dev")
+
+if APP_NAME:
+    USER_AGENT = "{app_name}/{package_version} (Kiwi.com {environment})".format(
+        app_name=APP_NAME, package_version=PACKAGE_VERSION, environment=ENVIRONMENT
+    )
+else:
+    USER_AGENT = None
+
+
+def set_user_agent(app_name=None, package_version=None, environment=None):
+    """Allows to customize or set User-Agent manually."""
+    app_name = app_name or APP_NAME
+    package_version = package_version or PACKAGE_VERSION
+    environment = environment or ENVIRONMENT
+    global USER_AGENT
```

Comment:
I think that with the way we're patching, we don't need to set a global for this at all. You could change this function to just return the `user_agent`, and then call it in `patch_with_user_agent()` to render the correct user-agent string specifically for the wrapper we're adding.